packages/builder/src/components/ContextMenu.svelte

```diff
@@ -0,0 +1,68 @@
+<script>
+  import { contextMenuStore } from "stores/builder"
+  import { Popover, Menu, MenuItem } from "@budibase/bbui"
+
+  let dropdown
+  let anchor
+
+  const showDropdown = dropdown => {
+    dropdown?.show()
+  }
+
+  $: showDropdown(dropdown)
```

Comment:
You can simplify showing the popover a little - there's a prop called `open` on the Popover component, so you can actually remove this `showDropdown` function, as well as the second `{#if ...}` statement around the popover, and instead just pass in `open={$contextMenuStore.visible}`.

Reply:
This was originally done because the popover wouldn't "notice" a change to it's provided anchor element, so the menu would show once then never again. I found a cleaner way to do it however, by doing as you suggested and putting the key statement around the anchor element itself.